test/regressions/TestViewer.js

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
+import clsx from 'clsx';
```

Comment:
why not use the `classnames` utils like everywhere else, so we avoid the dependency?